theory/DD/countpairs_impl.c.src

```diff
@@ -252,7 +276,7 @@ int countpairs_DOUBLE(const int64_t ND1, DOUBLE *X1, DOUBLE *Y1, DOUBLE *Z1,
     /*---Create 3-D lattice--------------------------------------*/
     int nmesh_x=0,nmesh_y=0,nmesh_z=0;
     cellarray_DOUBLE *lattice1 = gridlink_DOUBLE(ND1, X1, Y1, Z1, &(extra->weights0),
-                                                 xmin, xmax, ymin, ymax, zmin, zmax,
+                                                 xmin, xdiff, ymin, ydiff, zmin, zdiff,
```

Comment:
If not *absolutely* necessary, I would recommend against changing the semantics of function parameters.

Reply:
Better to add the `x/y/z/diff` parameters and change the function signature entirely